function.hpp

```diff
@@ -484,7 +484,7 @@ struct storage_t<signature<ReturnType(Args...)>, Qualifier, Config>
 
     vtable_ptr_t _vtable;
 
-    void* _impl;
+    void* _impl = nullptr;
```

Comment:
Yes, but I wanted the compiler to have a chance to set it asap on construction.
Not sure how to do it properly, but if it's still optimized just by calling tidy() then ok.